src/Confluent.Kafka/Timestamp.cs

```diff
@@ -27,6 +27,11 @@ namespace Confluent.Kafka
     public struct Timestamp
     {
         /// <summary>
+        ///     Unix epoch as UTC DateTime
+        /// </summary>
+        public static readonly DateTime UnixEpochUtcDateTime = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
`UnixTimeEpoch` is more correct i believe - this is a thing related to unix time, not a specific timestamp. The word epoch already covers that it is a specific time. I don't think there is any naming inconsistency.

I think it's better to exclude Utc from the name as this is part of the definition of what unix time is. Including it in the name to me might imply there is some other way this could have been reasonably specified (which there is not). Also, I don't think there is any likely confusion as this is in the summary commend and it's also a property on the DateTime instance. If I thought there was likely a scenario in which this was likely to trip people up, i'd have been in favor of including this in the name, but I think that's not the case (its possible I'm wrong on that).